From 0e30aa2e10cf635268a9ad703a4a2f23d380ef7d Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 9 Jan 2020 21:09:55 +0100
Subject: [PATCH 1/1] travis: build HTML docs

Several patches delivered incorrect restructured text as documentation.
We should be able to discover this in Travis CI.

Provide a build step for 'make htmldocs'.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 .travis.yml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.travis.yml b/.travis.yml
index 5da046ca7e..49339f1ac5 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -362,6 +362,10 @@ matrix:
     - name: "grep FIXME HACK"
       script:
         - grep -r FIXME .
+    # build HTML documentation
+    - name: "htmldocs"
+      script:
+        - make htmldocs
     # search for HACK within source tree and ignore HACKKIT board
       script:
         - grep -r HACK . | grep -v HACKKIT
-- 
2.24.1

